backend/routes/postReaction.py

```
@@ -264,4 +265,26 @@ def remove_rsvp(event_id: int, db: Session = Depends(get_db), current_user: User
 
     db.delete(rsvp)
     db.commit()
-    return {"message": "RSVP removed successfully"}
\ No newline at end of file
+    return {"message": "RSVP removed successfully"}
+
+@router.get("/event/{event_id}/my_rsvp/")
+def get_user_rsvp_status(event_id: int, db: Session = Depends(get_db),current_user: User =Depends(get_current_user)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_user_rsvp_status`

(ANN201)